pyFV3/stencils/__init__.py

```diff
@@ -0,0 +1,28 @@
+from .a2b_ord4 import AGrid2BGridFourthOrder
```

Comment:
Poking @oelbert : an idea here. If we put down some one/two liner to explicitly describe each Module scientific purpose it may look nice in the docs. To be tested.

Reply:
probably a good idea, something like?

`"""
Converts a field from the Arakawa A grid to B grid
"""`

Reply:
Yes, we have to generate the docs and see how it shows